src/test_typing_extensions.py

```diff
@@ -5267,6 +5270,10 @@ class Y(Protocol[T, P]):
                 self.assertEqual(G2.__args__, (int, Concatenate[int, P_2]))
```

Comment:
Should the `__args__` here be replaced by `get_args`?

Reply:
fine to keep it as it is, realistically this attribute is never going away :)